ormar/models/mixins/save_mixin.py

```diff
@@ -238,6 +239,21 @@ def populate_default_values(cls, new_kwargs: Dict) -> Dict:
                 new_kwargs.pop(field_name, None)
         return new_kwargs
 
+    @classmethod
+    def populate_onupdate_value(cls, new_kwargs: Dict) -> Dict:
+        """
+        Populate value which from onupdate options in field
+
+        :param new_kwargs: dictionary of model that is about to be saved
+        :type new_kwargs: Dict
+        :return: dictionary of model that is about to be saved
+        :rtype: Dict
+        """
+        for field_name, field in cls.Meta.model_fields.items():
+            if field.has_onupdate() and not field.pydantic_only:
+                new_kwargs[field_name] = field.get_onupdate()
+        return new_kwargs
```

Comment:
This should use the `get_fields_with_onupdate` defined below to remove repetition and utilize the cache

```python
for field_name in cls.get_fields_with_onupdate():
    new_kwargs[field_name] = cle.Meta.model_fields[field_name].get_onupdate()
return new_kwargs
```

Also, this does not allow explicit overwrites (if you want to manually overwrite the field, it will be changed to onupdate value anyway).